### PR TITLE
Update imu 16505 diag test and imu test

### PR DESCRIPTION
**Pull request by VHolonec**
Update imu 16505 diag test and imu test

### CODE REVIEW

File: test/src/imu_subscriber_test.cpp [hunk 1]
```diff
@@ -23,32 +23,59 @@
...
 {
 public:
   static void SetUpTestCase() {}
 
   static void TearDownTestCase() { rclcpp::shutdown(); }
+
 };
 
-TEST(ImuSubscriberTest, test_acceleration_pozitive_values1)
+TEST(ImuSubscriberTest, test_acceleration_and_gyroscope)
```
Comment (rbolboac) [inline test/src/imu_subscriber_test.cpp:37]: please rename this to test_imu_publisher

File: test/src/imu_subscriber_test.cpp [hunk 2]
```diff
@@ -58,32 +85,27 @@ TEST(ImuSubscriberTest, test_acceleration_pozitive_values1)
...
-  }
-
-  //executor.spin();
-
-  ASSERT_TRUE(true);
+  executor.spin_once(sec);
 }
 
-TEST(ImuSubscriberTest, test_acceleration_pozitive_values2)
+TEST(ImuSubscriberTest, test_acceleration)
```
Comment (rbolboac) [inline test/src/imu_subscriber_test.cpp:91]: I think this test can be removed, right?

File: test/src/velangtemp_subscriber_test.cpp [hunk 3]
```diff
@@ -35,35 +36,53 @@ class VelAngTempSubscriberTest : public ::testing::Test
 
 TEST(VelAngTempSubscriberTest, test_velangtemp_data_intervals)
```
Comment (rbolboac) [inline test/src/velangtemp_subscriber_test.cpp:37]: let's keep the same naming convention to all tests test_velangtemp_publisher

File: test/src/velangtemp_subscriber_test.cpp [hunk 4]
```diff
@@ -35,35 +36,53 @@ class VelAngTempSubscriberTest : public ::testing::Test
...
-    ASSERT_TRUE(msg.delta_vel.y >= -10000 && msg.delta_vel.y <= 10000);
-    ASSERT_TRUE(msg.delta_vel.z >= -10000 && msg.delta_vel.z <= 10000);
+    int32_t minint16 = -32768;
+    int32_t maxint16 = 32767;
 
-    ASSERT_TRUE(msg.delta_angle.x >= -10000 && msg.delta_angle.x <= 10000);
-    ASSERT_TRUE(msg.delta_angle.y >= -10000 && msg.delta_angle.y <= 10000);
-    ASSERT_TRUE(msg.delta_angle.z >= -10000 && msg.delta_angle.z <= 10000);
+    double maxRangeTemp = maxint16 * scale_temp / 1000.0;
+    double minRangeTemp = minint16 * scale_temp / 1000.0;
```
Comment (rbolboac) [inline test/src/velangtemp_subscriber_test.cpp:75]: let's add the / 1000.0 in the API returning temperature scale

**Review by rbolboac — COMMENTED** [review]
few remarks from my side, can be merged after they are resolved

**Review by rbolboac — APPROVED** [review]
lgtm

**Review by rbudai98 — APPROVED** [review]
lgtm

**Pull request merged**